entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -613,7 +613,8 @@ class EntityService(
     }
 
     fun updateEntityLastMeasure(observation: Observation) {
-        val observingEntity = neo4jRepository.getObservingSensorEntity(observation.observedBy, EGM_VENDOR_ID, observation.attributeName)

```

Comment:
If possible, remove this from the commit (that's the only diff in the file and it's only formatting)